src/state_machine.zig

```diff
@@ -759,7 +889,7 @@ pub fn StateMachineType(
             assert(input.len <= self.batch_size_limit);
 
             self.prepare_timestamp += switch (operation) {
-                .pulse => 0,
+                .pulse => constants.batch_max.create_transfers, // Max transfers to expire.
```

Comment:
~Was this a bug before?~ Ah I see -- previously weren't creating any new objects, we were only updating objects in-place. But now we are inserting "account events" with their own timestamps, which means the prepare/commit timestamps are now advanced by transfer expiry!